src/main/java/com/yahoo/identity/services/storage/sql/SqlAccount.java

```diff
@@ -0,0 +1,79 @@
+package com.yahoo.identity.services.storage.sql;
+
+import com.yahoo.identity.services.account.Account;
+import com.yahoo.identity.IdentityException;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+
+import javax.annotation.Nonnull;
+import java.time.Instant;
+
+public class SqlAccount implements Account {
+    private AccountModel account;
+
+    public SqlAccount(@Nonnull SqlSessionFactory sqlSessionFactory, @Nonnull String id) throws IdentityException {
```

Comment:
`SqlAccount` shouldn't do sql operation itself. It should take an instance of `AccountModel`.

Reply:
So should all sql operations be done at `SqlStorage` level and pass the `AccountModel` instance to `SqlAccount`, `SqlAccountCreate`, and `SqlAccountUpdate`? I did it like this because this is what `SqlAccountCreate` has done (doing sql inside of it instead of taking an instance of `AccountModel`). And `SqlAccountCreate` is created like this before we started working on anything.